doc/Install.md

```diff
@@ -53,7 +53,7 @@ The Linux commands to clone the repository into a directory "mywebsite" would be
 Make sure the folder *view/smarty3* exists and is writable by the webserver user
 
     mkdir view/smarty3
-    chmod 777 view/smarty3
+    chmod www-data:www-data view/smarty3
```

Comment:
As I stated before, `www-data` is a user/group but `chmod` is for access rights, like `0755` or `0644` et cetera. Please change to 0750 (or 0755) for better safety and correct usage.

Reply:
Indeed, nice catch.

Reply:
Indeed, and I just saw I used chmod instead of chown, sorry.

Even if we use 0755, it won't help if it's not owned by the webserver/fpm's user, am I right?

Reply:
Yes, you are right.

Reply:
The fpm procress must then run as the same user with `suexec`, then it *should* work.